File: csharp/src/PosIntegration.Service/Est/Model/EstRefundResponse.cs
namespace PosIntegration.Service.Est.Model
{
    public class EstRefundResponse
    {
        public string Response { get; set; }
        public string ErrorCode { get; set; }
        public string Message { get; set; }
        public string RawBody { get; set; }

        public bool IsSuccess()
        {
            return "[refund-ok]".Equals(Response);
        }
    }
}

File: java/src/main/java/com/pos/integration/service/est/model/EstRefundResponse.java
package com.pos.integration.service.est.model;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class EstRefundResponse {

    private final String response;
    private final String errorCode;
    private final String message;
    private final String rawBody;

    public boolean isSuccess() {
        return "[refund-ok]".equals(response);
    }
}
